src/logic/textBox.js

```diff
@@ -1,34 +1,28 @@
-import { facts, jokes, answers } from "./textBoxData";
 import {
   addData,
   displayStoredData,
 } from "./localStorage";
-import axios from 'axios';
+import { sendMessage, jokesAndFacts, chatResponse} from "./chatGPT";
 
-const getTextBoxData = (data) => {
-  const randomNumber = Math.floor(Math.random() * data.length);
-  return data[randomNumber];
-};
 
 const displayTextBox = (text) => {
   const paragraph = document.createElement("div");
   const answerOutput = document.getElementById("answerOutput");
 
   paragraph.innerHTML = text;
-
   paragraph.classList.add("textBox");
 
   answerOutput.appendChild(paragraph);
 };
 
-const showAnswerForQuestion = (question, answer) => {
+const showAnswerForQuestion = (question, chatResponse) => {
   const answerOutput = document.getElementById("answerOutput");
   const questionParagraph = document.createElement("p");
   questionParagraph.textContent = question;
   questionParagraph.classList.add("question-text");
 
   const answerParagraph = document.createElement("p");
-  answerParagraph.textContent = answer;
+  answerParagraph.textContent = chatResponse;
```

Comment:
pretty nice change. you keep the code more readable as other programmer is able to see exactly what you mean by answer and don't have to look for this.